types/forge-viewer/index.d.ts

```diff
@@ -939,27 +940,27 @@ declare namespace Autodesk {
         }
 
         class ToolInterface {
```

Comment:
I'm not sure I see where where this is changed?

https://forge.autodesk.com/en/docs/viewer/v7/reference/Viewing/ToolInterface/

Reply:
Some methods of the class are optional.